core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -193,6 +193,7 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
   // Static membership mapping [key: group.instance.id, value: member.id]
   private val staticMembers = new mutable.HashMap[String, String]
   private val pendingMembers = new mutable.HashSet[String]
+  private val MEMBER_ID_DELIMITER = "-"
```

Comment:
nit: we can put this in the companion object since it is a constant. Usual convention in scala is to use capitalized camel case for constants (e.g. `MemberIdDelimiter`).

Reply:
Ah good catch, I missed this part.